Firestore/core/src/firebase/firestore/remote/datastore.mm

```diff
@@ -292,6 +308,48 @@ void LogGrpcCallFinished(absl::string_view rpc_name,
   active_calls_.erase(found);
 }
 
+bool Datastore::IsAbortedError(const Status& error) {
+  return error.code() == FirestoreErrorCode::Aborted;
+}
+
+bool Datastore::IsPermanentError(const Status& error) {
+  switch (error.code()) {
+    case FirestoreErrorCode::Ok:
+      HARD_FAIL("Treated status OK as error");
+      return true;
```

Comment:
This line's unreachable. Unless the linter's complaining, or you're trying to maintain consistency with the other ports, you could probably just drop it. Optional.

Reply:
Done.